### DISCUSSION TITLE
Small issue in csproj

### DISCUSSION THREAD

**Discussion by jaime-olivares** [fo-dicom/fo-dicom#1888] [General]
I don't want to create a new issue for such a small one:

The following lines are duplicated package references. They should be removed:
https://github.com/fo-dicom/fo-dicom/blob/aced2b6ac2e15692aaeccbde5d4e4f007b9830d6/Tests/FO-DICOM.Benchmark/FO-DICOM.Benchmark.csproj#L13

<img width="559" alt="image" src="https://github.com/user-attachments/assets/3e31b830-03e9-4914-a43a-9173e871b92e">

**Comment by mrbean-bremen** [comment]
Just make a small PR?

**Comment by jaime-olivares** [comment]
Yeah, that's better. I have created it here: https://github.com/fo-dicom/fo-dicom/pull/1889
